### PR TITLE
[Python] fix SparseDataFrames exception

### PR DESCRIPTION
**Pull request by jorahn**
test for sparse dataframe #221

### CODE REVIEW

**Comment by jorahn** [comment]
added the fix in api.py to the same pr as the test

File: python/feather/api.py [hunk 1]
```diff
@@ -27,6 +27,9 @@ def write_dataframe(df, path):
     Write a pandas.DataFrame to Feather format
     '''
     writer = ext.FeatherWriter(path)
+    
+    if isinstance(df, pd.sparse.frame.SparseDataFrame):
```
Comment (wesm) [inline python/feather/api.py]: Can you use `pd.SparseDataFrame` here?

**Comment by jorahn** [comment]
sure

**Comment by wesm** [comment]
Thank you!

**Pull request closed without merging**